releasenotes/notes/max_stack_trace_depth_top_percent-a840b8487674c046.yaml

```diff
@@ -0,0 +1,11 @@
+---
+upgrade:
+  - |
+    ASM: With this upgrade, you can now control how the stack trace report are cropped when reported for exploit prevention or IAST.
+      - DD_APPSEC_MAX_STACK_TRACE_DEPTH allowed to control the maximum stack trace size reported (default 32)
+      - DD_APPSEC_MAX_STACK_TRACE_DEPTH_TOP_PERCENT allows now to specify how the stack trace is cropped as a percentage.
```

Comment:
If these are configurable by customers, can we add these to the configuration.rst doc?

Reply:
I'll do it now.